### ISSUE TITLE
Signing functionality should handle duplicates in signing.props

### ISSUE THREAD

**Issue by missymessa** [dotnet/arcade#5939]
- [ ] This issue is blocking 
- [ ] This issue is causing unreasonable pain

Since the signing functionality imports a customer created `signing.props` file into the default `sign.props` file, it should be responsible for handling duplicate values that are referenced in `signing.props`. The customer should not have to have knowledge about how `sign.props` is configured. Should something in `sign.props` change that would affect previously built `signing.props`, then customers would break. 

See this PR for reasons we had to add workarounds in the resulting `signing.props` to account for collisions with data in `sign.props`: https://dev.azure.com/dnceng/internal/_git/dotnet-release/pullrequest/9678?_a=files&discussionId=46249&path=%2Fsrc%2FMicrosoft.DotNet.Release%2FMicrosoft.DotNet.Signing.Extensions%2FOperations%2FCreateSigningPropsOperation.cs

**Comment by missymessa** [comment]
This issue is not specific to Post Build Signing. I don't think this is an appropriate epic for this issue to be in.

**Comment by riarenas** [comment]
The rationale is that this is the current epic for signing functionality. Which epic do you think is more appropriate for this?

**Comment by missymessa** [comment]
> The rationale is that this is the current epic for signing functionality. Which epic do you think is more appropriate for this?

An epic that encompasses tech debt in Arcade, which probably doesn't exist.

**Comment by chcosta** [comment]
@missymessa , can we close this?

**Comment by missymessa** [comment]
> @missymessa , can we close this?

I don't think it should be in our epic. It should be in a tech debt epic for Arcade (I don't think one exists yet). Alternatively, this may be a moot issue once no one is using the signing functionality in this way.

**Comment by chcosta** [comment]
I'm not sure what the issue is here.  Given that we now handle signing metadata which allows collision resolution based on bar build id, can we close this?

**Comment by missymessa** [comment]
This isn't an issue with Post Build Signing. This is an issue with having to set up Signing.props to handle collisions for the original usage of Sign.proj. As I mentioned, this should be in an epic to deal with Arcade Tech Debt.
